src/ui/public/vislib/__tests__/lib/layout/layout.js

```diff
@@ -52,22 +56,26 @@ dateHistogramArray.forEach(function (data, i) {
     describe('createLayout Method', function () {
       it('should append all the divs', function () {
         expect($(vis.el).find('.vis-wrapper').length).to.be(1);
-        expect($(vis.el).find('.y-axis-col-wrapper').length).to.be(1);
+        expect($(vis.el).find('.y-axis-col-wrapper').length).to.be(2);
         expect($(vis.el).find('.vis-col-wrapper').length).to.be(1);
-        expect($(vis.el).find('.y-axis-col').length).to.be(1);
-        expect($(vis.el).find('.y-axis-title').length).to.be(1);
-        expect($(vis.el).find('.y-axis-div-wrapper').length).to.be(1);
-        expect($(vis.el).find('.y-axis-spacer-block').length).to.be(1);
+        expect($(vis.el).find('.y-axis-col').length).to.be(2);
+        expect($(vis.el).find('.y-axis-title').length).to.be(2);
+        expect($(vis.el).find('.y-axis-div-wrapper').length).to.be(2);
+        expect($(vis.el).find('.y-axis-spacer-block').length).to.be(4);
         expect($(vis.el).find('.chart-wrapper').length).to.be(numberOfCharts);
-        expect($(vis.el).find('.x-axis-wrapper').length).to.be(1);
-        expect($(vis.el).find('.x-axis-div-wrapper').length).to.be(1);
-        expect($(vis.el).find('.x-axis-title').length).to.be(1);
+        expect($(vis.el).find('.x-axis-wrapper').length).to.be(2);
+        expect($(vis.el).find('.x-axis-div-wrapper').length).to.be(2);
+        expect($(vis.el).find('.x-axis-title').length).to.be(2);
```

Comment:
i don't see an easy way to do it ... but i wouldn't worry about layout class that much as i intend to remove it in 5.3 .... currently we use a combination of HTML and SVG to render charts ... i don't like that, i think it should all be a single SVG element. One direct benefit of that is that it will allow us to save all charts as images. Also i think the code will be clearer and easier to read.